packages/lib/src/Button.wc.svelte

```diff
@@ -12,9 +14,10 @@
   <a {href} class="button" class:outline {onclick}><slot /></a>
 {:else}
   <button
-    class="button bg-yellow-500"
+    class="button bg-yellow-500 {size}"
```

Comment:
Guess there might be a blank space in here which probably doesn't matter, but..

Reply:
If we decide on committing to tailwind-merge it solves it for us